client/reader/controller.js

```diff
@@ -59,8 +60,14 @@ function removeFullPostDialog() {
 	__fullPostInstance = null;
 }
 
-function ensureStoreLoading( store ) {
+function ensureStoreLoading( store, context ) {
```

Comment:
Do we need to hand in the entire `context` here, or would `context.query` be enough?